version/distro/distro.go

```diff
@@ -147,3 +152,28 @@ func DSMVersion() int {
 		return v
 	})
 }
+
+func isUDMPro() bool {
+	if runtime.GOOS != "linux" {
```

Comment:
Even though this isn't strictly necessary since it's currently only called from `linuxDistro()`, it feels safer to keep it. I can remove it if you feel it's unnecessary overhead.